website/docs/robots.txt

```diff
@@ -1,3 +0,0 @@
-User-agent: *
-Allow: /
-Sitemap: https://docs.luigi-project.io/sitemap.xml
```

Comment:
I think this file should not be deleted. Should help with SEO optimisation on server side or sth similar. Please revert accordingly